wiki/markdown/running-eiffel-intelligence.md

```diff
@@ -41,8 +50,43 @@ remember to keep the name of the properties file if you are a beginner to
 Spring. More advanced Spring user can look [here](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html)
 for more information about using external configuration.
 
-If only few properties need to be overridden then use java opts, for example
+If only few properties need to be overridden, then use java opts, for example:
 
     java -jar eiffel-intelligence-<version>.war -Dspring.data.mongodb.port=27019
 
-All available Eiffel-Intelligence properties can be found in [application.properties](https://github.com/Ericsson/eiffel-intelligence/blob/master/src/main/resources/application.properties) example file.
\ No newline at end of file
+## Running in Tomcat instance
+
+To run Eiffel-Intelligence in Tomcat, the war file must be put into the webapp folder in tomcat installation folder, also called catalina home folder:
+
+    (catalina home)/webapp/
+
+If Eiffel-Intelligence should be run without any conext-path in the url address, then overwrite ROOT.war file in webapp folder with eiffel-intelligence-<version>.war:
+
+    cp eiffel-intelligence-<version>.war (catalina home)/webapp/ROOT.war
+
+Remove "ROOT" folder in webapp folder:
+
+    rm -rf (catalina home)/webapp/ROOT/
+
+Create "config" folder in webapp folder, if it not exists. Spring and Eiffel-Intelligence will look for the application.properties configuration file in config folder:
+
+    mkdir (catalina home)/webapp/config
+
+Copy the application.properties file into the newly created config folder:
+    
+    cp application.properties (catalina home)/webapp/config
+
+Start Tomcat and Eiffel-Intelligence in background/daemon mode by exectuing command:
+
+    (catalina home)/bin/catalina.sh start
+
+To run Tomcat and Eiffel-Intelligence with logs printed to console:
+    
+    (catalina home)/bin/catalina.sh run
+
+## Eiffel-Intelligence configurations and properties
+
+All available Eiffel-Intelligence properties can be found in [application.properties](https://github.com/Ericsson/eiffel-intelligence/blob/master/src/main/resources/application.properties) example file.
+
+More documentation of each Eiffel-Intelligence properties and configurations can be found in [Configuration page](./configuration.md)
```

Comment:
I'm not usre the relative link will work in github pages, so keep full path to the github file instead.

Reply:
fixed.